recipe/meta.yaml

```diff
@@ -21,10 +24,13 @@ requirements:
 
   run:
     - python
+    - setuptools
```

Comment:
`setuptools` is imported by `main.py`: https://github.com/timothycrosley/isort/blob/develop/isort/main.py#L28